source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -253,6 +253,75 @@ const callback = filename => (err, data) => {
 };
 ```
 
+#### Use of `async`/`await`
+
+If your version of Node is above 7.10.0 you should use async functions, and the await keyword, to organise your asynchronous code. This will make it easier to read and write.
+
+```js
+// without async/await
+const getUserFromCreds = function (username, password) {
+  return getUserIdFromAuth(username, password)
+    .then(userId => {
+      return getUserFromApi(userId);
+    })
+    .then(user => {
+      return {
+        'id': user.id,
+        'name': user.name,
+        'age': user.ageInYears      
+      };
+    })
+    .catch(e => {
```

Comment:
I don't think having the `.catch` and `try/catch` logic in these examples is required to demonstrate the main differences here

having a try/catch with an console.log would probably be considered bad practice within a async function (should always let the error propagate up to the caller) so the code ends up looking a artificial in the second example.

If you want talk about how error handling changes, maybe it deserves it's own example

Reply:
Makes sense. I added it to ape the examples in https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await but I see your point. Now removed.

Would you be interested in raising another PR to add a section on error handling in asynchronous JS?